telco-core/configuration/reference-crs/optional/cert-manager/consoleCertificate.yaml

```diff
@@ -0,0 +1,21 @@
+---
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: console-certificate
+  namespace: openshift-console
+  annotations:
+    argocd.argoproj.io/sync-wave: "-30"
+    argocd.argoproj.io/sync-options: SkipDryRunOnMissingResource=true
+spec:
+  commonName: selfsigned-ca.dns.name
+  isCA: false
+  issuerRef:
+    group: cert-manager.io
+    kind: ClusterIssuer
+    name: acme-issuer
+  privateKey:
+    algorithm: ECDSA
+    size: 256
+  secretName: console-ca-root-secret
```

Comment:
Rather than a cert for the console endpoint the reference should obtain a wildcard cert for ingress *.apps.<fqdn>. This would cover console and all other ingress endpoints. This is now possible with the dns challenge.